tests/_craft/migrations/m220617_180419_add_fields.php

```diff
@@ -61,21 +58,33 @@ public function safeUp()
         $groupId = (int)$group['id'];
 
         $fieldLayout = $entryType->getFieldLayout();
-        $tab = $fieldLayout->getTabs()[0];
-
-        $newFields = [
-            $this->createPlaintextField($groupId),
-            $this->createRedactorField($groupId),
-            $this->createMatrixField($groupId),
-            $this->createCheckboxesField($groupId),
-            $this->createLightswitchField($groupId),
-            $this->createSuperTableField($groupId),
-            $this->createTableField($groupId),
-            $this->createNeoField($groupId),
-        ];
+        $tabs = $fieldLayout->getTabs();
+
+        $newFields[] = $this->createPlaintextField($groupId);
+        $newFields[] = $this->createRedactorField($groupId);
+        $newFields[] = $this->createMatrixField($groupId);
+        $newFields[] = $this->createCheckboxesField($groupId);
+        $newFields[] = $this->createLightswitchField($groupId);
+
+        if (TEST_SUPERTABLE_PLUGIN) {
```

Comment:
where does `TEST_SUPERTABLE_PLUGIN` come from and what behavior does it control?

Reply:
It is coming from [bootsrap.php](https://github.com/lilt/craft-lilt-plugin/pull/17/files#diff-f43957ff1990ab850481d9be76abbd0f6c804db95704fec5c901bea250e52eb6R16), it is used to disable some test functionality. 

Since supertable, linkit and colourswatches not avialable for Craft V4, we can't test integration of them